powerdns/models/powerdns.py

```diff
@@ -723,3 +727,39 @@ class Meta:
 
     def __str__(self):
         return self.domain
+
+
+
+try:
+    import pyhermes
+except ImportError:
+    pass
+else:
+    if settings.TXT_AUTOUPDATE_TOPIC_NAME:
+        @transaction.atomic
+        @pyhermes.subscriber(topic=settings.TXT_AUTOUPDATE_TOPIC_NAME)
+        def _update_auto_txt(data):
```

Comment:
imo it could be used not only for txt records, but for other types (especially A) as well.